tests/bson/bson-symbol-set_state-001.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+MongoDB\BSON\Symbol::__set_state()
+--FILE--
+<?php
+
+var_export(MongoDB\BSON\Symbol::__set_state([
+	'symbol' => "symbolValue",
```

Comment:
Tab alert.

Reply:
![](https://pbs.twimg.com/profile_images/824647153721106432/gwLIQxqy.jpg)

Reply:
@derickr: I'll see your :-1: and raise you a:

![](https://i.pinimg.com/736x/41/dc/4e/41dc4e8deaa4fece85a3895aea8c74a8--indian-engagement-in-india.jpg)